actions.go

```diff
@@ -3,10 +3,123 @@ package neo4j
 import (
 	"fmt"
 
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
 	"github.com/cloudprivacylabs/opencypher/graph"
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 )
 
+type neo4jAction interface {
+	Run(neo4j.Transaction, map[graph.Node]int64) error
+}
+
+var neo4jActions = map[string]neo4jAction{
+	"Address":     recreate{},
+	"Observation": insert{},
+}
+
+type recreate struct {
+	Config
+	graph.Graph
+	graph.Node
+	entityId int64
+}
+
+func (r recreate) Run(tx neo4j.Transaction, nodeIds map[graph.Node]int64) error {
```

Comment:
Instead of recreate, just define a deleteEntity, and insertEntity. Recreate is delete-insert.